_posts/2015-12-15-global-warming

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title:  "Global Variables"
+date:   2015-12-15 01:49:56
+categories: nodejs javascript
+author:
+ twitter: sahilbathla
+ name: Sahil Batla
```

Comment:
You can add your name and twitter handle in the `authors` list present in the `config.yml` file
From there you can add them here with `authorslug`